extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -349,6 +353,12 @@ export class AmpSlideScroll extends BaseSlides {
       return Promise.resolve();
     }
 
+    // Account for race when onResized_ has not fired before layoutCallback,
+    // since we need slideWidth_ to proceed.
+    if (!this.hasFirstResizedOccured_ && this.slidesContainer_) {
```

Comment:
`this.slidesContainer_` should be defined in `buildCallback`, right? 
```suggestion
    if (!this.hasFirstResizedOccured_) {
```

Reply:
I wanted to be extra safe, but you're right.